src/lib/protocols/credentials/__tests__/CredentialService.test.ts

```diff
@@ -286,15 +286,22 @@ describe('CredentialService', () => {
     });
 
     test('returns credential request message base on existing credential offer message', async () => {
-      const credentialRequest = await credentialService.createCredentialRequest(connection, credentialRecord, credDef);
+      // given
+      const comment = 'credential request comment';
+
+      // when
+      const credentialRequest = await credentialService.createCredentialRequest(connection, credentialRecord, credDef, {
+        comment,
+      });
```

Comment:
Also 4 parameters is a lot I think, wouldn't it be better to make it a single object? This is not for this PR, but just a thought.

```suggestion
      const credentialRequest = await credentialService.createCredentialRequest({
          connection, 
          credentialRecord, 
          credentialDefinition: credDef,
          comment,
      });
```